docs/cdp/cdp_data_customization.md

```diff
@@ -0,0 +1,99 @@
+By default, custom Field Types have basic export functionality. It casts their `Value` object to string, thanks to `\Stringable` implementation.
+​
+## Exporting Field Types
+​
+Field Types are exported with metadata, for example, ID, Field Definition name, type, value. 
+You can also provide your own  `\Ibexa\Contracts\Cdp\Export\Content\FieldProcessorInterface` instance to extend metadata. 
+Provided implementation has to be defined as a service and tagged with `ibexa.cdp.export.content.field_processor`. 
+Additionally, you can specify `priority` to override default behavior. 
+All system Field Processors use `-100` priority, and any higher priority value overrides them.
+
+The interface is plain and has two methods that you need to provide:
+
+- **supports** - decides whether your `FieldProcessor` can work with the `Field` instance.
+- **process** - takes `Field` instance and then returns a flat array of scalar values that are combined with the payload data.
+​
+A common Field Type is serialized to:
+​
+```json
+{
+    "field_measurement_simple_id": 1792,
+    "field_measurement_simple_type": "ibexa_measurement",
+    "field_measurement_simple_language_code": "eng-GB",
+    "field_measurement_simple_value_measurement": "data transfer rate",
+    "field_measurement_simple_value_unit_identifier": "megabyte per second",
+    "field_measurement_simple_value_unit_symbol": "MB/s",
+    "field_measurement_simple_value_unit_is_base": false,
+    "field_measurement_simple_value_base_unit_identifier": "bit per second",
+    "field_measurement_simple_value_base_unit_symbol": "bit/s",
+    "field_measurement_simple_value_simple": 100,
+    "field_measurement_simple_value_simple_base_unit": 800000000
+}
+```
+​
+Field identifier is a prefix that is automatically added to each key. You can only use scalar values.
+​
+### Built in Field Processors for custom Field Types
+​
+You may provide your own CDP export functionality by using one of the system Field Processors:
+
+#### `\Ibexa\Cdp\Export\Content\FieldProcessor\SkippingFieldProcessor`.
+​
+It results in the Field Type being excluded from the exported payload.
+To avoid adding the Field Type data to the payload, register a new service as follow:
+​
+```yaml
+custom_fieldtype.cdp.export.field_processor:
+    class: Ibexa\Cdp\Export\Content\FieldProcessor\SkippingFieldProcessor
+    autoconfigure: false
+    arguments:
+        $fieldTypeIdentifier: custom_fieldtype
+    tags:
+        - { name: 'ibexa.cdp.export.content.field_processor', priority: 0 }
+```
+​
+## Exporting Field Type values
+​
+To customize export of Field Type values, provide your own `\Ibexa\Contracts\Cdp\Export\Content\FieldValueProcessorInterface` instance.
+New implementation has to be registered as a service manually or by using autoconfiguration. 
+The service has to use the tag `ibexa.cdp.export.content.field_value_processor`, you can also provide `priority` property to override other Field Value Processors.
```

Comment:
```suggestion
The service has to use the tag `ibexa.cdp.export.content.field_value_processor`. 
You can also provide the `priority` property to override other Field Value Processors.

```
I think a line is needed here, as the following bullets are not rendered properly.